packages/maskbook/src/plugins/Trader/UI/trending/CoinMetadataTable.tsx

```diff
@@ -46,11 +47,9 @@ const useStyles = makeStyles((theme) =>
                 paddingRight: 0,
             },
         },
-        tag: {
-            paddingRight: theme.spacing(2),
-            '&:last-child': {
-                paddingRight: 0,
-            },
+        tagCell: {
+            display: 'flex',
```

Comment:
Suppose you haven't changed the `display` of the table cell. It's unnecessary to set the margin-bottom property.